index.html

```diff
@@ -55,7 +54,140 @@ <h4>I'm Azeezat</h4>
           <a href="#"><i class="fab fa-medium"></i></a>
         </li>
       </ul>
-    <div id="background"><img src="image/Group.png" alt=""></div>
+      <div id="background"><img src="image/Group.png" alt="" /></div>
     </section>
+    <header class="work-section">
+      <h1><a href="#">My Recent Works</a></h1>
+    </header>
+
+    <hr class="solid" />
+
+    <div class="picture"><img src="image/Image(1).png" alt=""/></div>
+    <div class="works">
+      <p class="head">Multiple-Post Stories</p>
+      <p class="daily">
+        A daily selection of privately Personalized Reads; no account or signup
+        required has been the industry standard dummy text ever since 1500s,
+        when an unknown printer took a dummy text.
+      </p>
+    </div>
+    <div class="grid">
+      <ul class="grid-container">
+        <li class="grid-item">Css</li>
+        <li class="grid-item">Html</li>
+        <li class="grid-item">bootstrap</li>
+        <li class="grid-item">Ruby</li>
+      </ul>
+    </div>
+    <button id="button" type="submit">See Project</button>
+
+    <div class="bottom">
+      <a href="#"
+        ><img src="image/image.png" alt=""/></a><button class="see" type="submit">
+          See Project</button>
+      <a href="#"><img src="image/image.png" alt="" /></a><button class="see" type="submit">See Project</button>
+      <a href="#"><img src="image/image.png" alt=""/></a
+      ><button class="see" type="submit">See Project</button>
+      <a href="#"
+        ><img src="image/image.png" alt="" /></a
+      ><button class="see" type="submit">
+          See Project
+        </button>
+      <a href="#"
+        ><img src="image/image.png" alt="" /></a
+      ><button class="see" type="submit">
+          See Project
+        </button>
+    </div>
+    <br />
+    <br />
+    <section class="about">
+      <h2>About Me</h2>
+      <h3>
+        I'm a software Developer! I can help you build a product,feature or
+        website. Look through some of my work and Experience. if you like what
+        you see and have a project dont hesitate to contact me.
+      </h3>
+      <button id="Resume" type="submit">Get My Resume</button>
+
+      <div id="ground"><img src="Group2.png" alt=""/></div>
+
+      <hr class="solid" />
+
+      <div class="languages">
+        Language
+        <ul>
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+      </div>
+
+      <div class="framework">
+        Frameworks
+        <ul>
+          <li>Bootstraps</li>
+          <li>Ruby on Rails</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+      </div>
+
+      <div class="skill">
+        Skills
+        <ul>
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
```

Comment:
- [ ] Please wrap your **'Languages'**, **'Frameworks'** and **'Skills'** categories in a `ul` with each of them in a `li` tag

![langs](https://user-images.githubusercontent.com/36542986/134589830-1b61230c-13dd-48d8-87f1-039102a15de9.jpg)